odom/odom/config/localization.yaml

```diff
@@ -0,0 +1,52 @@
+frequency: 50
+
+# Specify planar environment
+two_d_mode: true
+
+map_frame: map
+odom_frame: caffeine/odom
+base_link_frame: caffeine/base_link
+world_frame: caffeine/odom
+
+# publish transform: world_frame --> base_link_frame
+publish_tf: true
+
+transform_time_offset: 0.05
+
+# See /diagnostics topic for debug
+print_diagnostics: true
+
+## Odometry Sensor(s) ##
+
+odom0: caffeine/odom
+odom0_config: [true,  true,  false,
+               false, false, true,
```

Comment:
I don't think `caffeine/odom` should use yaw (yaw_vel I'm ok with). This is because the IMU would provide appropriate yaw measurements since it also uses a magnetometer for a world reference frame, and so we don't want to have conflicting values for yaw